### DISCUSSION TITLE
Allow project owners to create webhooks

### DISCUSSION THREAD

**Discussion by Arbee4ever** [modrinth/.github#78] [Feature requests]
**Describe the solution you'd like**
Allow project owners to create webhooks for their projects, that they can then use to send messages into a discord channel whenever it updates.

**Describe alternatives you've considered**
There might be one, but I don't know...

**Additional context**
I would like to use webhooks to automatically send messages in my discord to notify players, that I updated the mod.

**Comment by triphora** [comment]
Doesn't even need to be restricted to project owners. This would basically be an extension of the Follow system, just also pushing to webhooks, not just to users' notifications.

**Comment by Arbee4ever** [comment]
I just said owners, because GitHub also only allows owners to do that.
